website/docs/r/management_group.html.markdown

```diff
@@ -39,9 +39,13 @@ resource "azurerm_management_group" "example_child" {
 
 The following arguments are supported:
 
-* `group_id` - (Optional) The UUID for this Management Group, which needs to be unique across your tenant - which will be generated if not provided. Changing this forces a new resource to be created.
+* `name` - (Optional) The name or UUID for this Management Group, which needs to be unique across your tenant. A new UUID will be generated if not provided. Changing this forces a new resource to be created.
 
-* `display_name` - (Optional) A friendly name for this Management Group. If not specified, this'll be the same as the `group_id`.
+* `group_id` - (Optional) The name or UUID for this Management Group, which needs to be unique across your tenant. A new UUID will be generated if not provided. Changing this forces a new resource to be created.
+
+~> **NOTE:** The field `group_id` has been deprecated in favor of `name`.
```

Comment:
We typically can just leave out deprecated properties as unless a user is using them they don't really need to know, and if they are the deprecation message will let them know the new proeprty

Reply:
You mean to remove this line of deprecation notice? Removed.